dex-it-common/src/main/scala/com/wavesplatform/dex/it/api/dex/DexApiOps.scala

```diff
@@ -32,14 +32,18 @@ object DexApiOps {
     def cancel(owner: KeyPair, order: Order): F[MatcherStatusResponse]                       = cancel(owner, order.assetPair, order.id())
     def cancel(owner: KeyPair, assetPair: AssetPair, id: Order.Id): F[MatcherStatusResponse] = explicitGet(self.tryCancel(owner, assetPair, id))
 
-    def cancelWithApiKey(order: Order): F[MatcherStatusResponse] = cancelWithApiKey(order.id())
-    def cancelWithApiKey(id: Order.Id): F[MatcherStatusResponse] = explicitGet(self.tryCancelWithApiKey(id))
+    def cancelWithApiKey(order: Order, xUserPublicKey: Option[PublicKey] = None): F[MatcherStatusResponse] =
+      cancelWithApiKey(order.id(), xUserPublicKey)
+    def cancelWithApiKey(id: Order.Id, xUserPublicKey: Option[PublicKey]): F[MatcherStatusResponse] =
+      explicitGet(self.tryCancelWithApiKey(id, xUserPublicKey))
```

Comment:
Please remind me, why do we need this `xUserPublicKey`?

Reply:
See one of the latest comments in the JIRA task